### PR TITLE
[GEOS-5391] GetCapabilities 1.1.1 and 1.3.0 does not include GetMap formats for exceptions

### PR DESCRIPTION
**Pull request by alex3305**
Added the missing application/vnd.ogc.se_blank exception that was missing from the valid exceptions for WMS 1.1.1. This exception has been added to the GetCapabilities, but also produces a valid, blank image when an exception is thrown. Also added tests for both the Capbilities and the exception handling, where the exception is compared to a blank image.

See https://jira.codehaus.org/browse/GEOS-5391

### CODE REVIEW

**Comment by jodygarnett** [comment]
Hey @alex3305 - this looks pretty darn good. Can I ask if you have signed a code contribution agreement for OSGeo? (See http://www.osgeo.org/content/foundation/legal/licenses.html).

GitHub should of asked you about https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md

**Comment by alex3305** [comment]
Yes, I've signed the contribution agreement for our company Geodan.

**Comment by jodygarnett** [comment]
Thanks, reviewing now.

File: src/wms/src/main/java/org/geoserver/wms/WMSServiceExceptionHandler.java [hunk 1]
```diff
@@ -163,13 +163,13 @@ public void handleServiceException(ServiceException exception, Request request)
     
     private boolean isImageExceptionType(String exceptions) {
         return "application/vnd.ogc.se_inimage".equals(exceptions) || "INIMAGE".equals(exceptions)
-                || "BLANK".equals(exceptions);
+                || "BLANK".equals(exceptions) || "application/vnd.ogc.se_blank".equals(exceptions);
```
Comment (jodygarnett) [inline src/wms/src/main/java/org/geoserver/wms/WMSServiceExceptionHandler.java]: Not sure I understand why "vnd.ogc.se_blank" is a fun exception type?

File: src/wms/src/main/java/org/geoserver/wms/WMSServiceExceptionHandler.java [hunk 2]
```diff
@@ -184,7 +184,7 @@ private void handleImageException(ServiceException exception, Request request, f
         g.setColor(bgcolor);
         g.fillRect(0, 0, img.getWidth(), img.getHeight());
         
-        if (!"BLANK".equals(exceptionFormat)) { //wms 1.3 only
```
Comment (jodygarnett) [inline src/wms/src/main/java/org/geoserver/wms/WMSServiceExceptionHandler.java]: Okay I got a question on this one - our user manual says:


Format	Syntax	Notes
Blank	EXCEPTIONS=application/vnd.ogc.se_blank		Generates a blank image

It looks like we are writing text onto our blank images - not sure if this is intended.

**Comment by jodygarnett** [comment]
@alex3305 or @aaime if you can answer my question above this pull request is good to go (in this case the docs are already written).

File: src/wms/src/main/java/org/geoserver/wms/WMSServiceExceptionHandler.java (hunk 1 above)
Comment (alex3305) [inline src/wms/src/main/java/org/geoserver/wms/WMSServiceExceptionHandler.java]: You'll have to specify this type of exception, if you don't want to get an error message displayed in the map composition (for a getMap request).

File: src/wms/src/main/java/org/geoserver/wms/WMSServiceExceptionHandler.java (hunk 2 above)
Comment (alex3305) [inline src/wms/src/main/java/org/geoserver/wms/WMSServiceExceptionHandler.java]: AFAIK GeoServer isn't writing text into blank images. The application/vnd.ogc.se_blank WMS 1.1.1 type is simply added to the original implementation for a blank exception.

**Comment by aaime** [comment]
Merged

**Pull request merged**